examples/dashboard-streamlit/README.md

```diff
@@ -0,0 +1,116 @@
+# Create A Dashboard on Saturn Cloud Using Streamlit
+
+![Streamlit logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/streamlit-logo.png "doc-image")
```

Comment:
We probably want to have a picture of the actual dashboard at the beginning, which maybe should be this image or like, the second image